CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [Unreleased](https://github.com/ScoopInstaller/Scoop/compare/master...develop)
```

Comment:
Since v0.5.3 has not been released yet, I think the log entry should be added under v0.5.3.